### PR TITLE
Change frontend routes to remove PDFType and old PDF Routes

### PR DESCRIPTION
**Pull request by steffen12**
## Description of changes
- Integrate new file and pdf routes
- Remove all usages of old pdf routes
- Remove all usages of PDFType enum on frontend

### CODE REVIEW

**Review by crchong1 — APPROVED** [review]

File: src/serverOverride.ts [hunk 1]
```diff
@@ -1,11 +1,14 @@
 const getServerURL = (): string => {
   if (process.env.REACT_APP_KEEPID_BUILD_ENV === 'staging') {
+    console.log('https://staged.keep.id');
```
Comment (crchong1) [inline src/serverOverride.ts:3]: do we need to keep this?

File: src/components/Documents/UploadDocs.tsx [hunk 2]
```diff
@@ -155,15 +154,15 @@ class UploadDocs extends React.Component<Props, State> {
         this.state.documentTypeList &&
         this.state.documentTypeList.length < this.state.pdfFiles.length
       ) {
-        this.props.alert.show('Please categorize each document');
-        error = true;
+        // this.props.alert.show('Please categorize each document');
+        // error = true;
```
Comment (crchong1) [inline src/components/Documents/UploadDocs.tsx:158]: why are we commenting out the error here?

File: src/components/Documents/MyDocuments.tsx [hunk 3]
```diff
@@ -174,9 +189,21 @@ class MyDocuments extends Component<Props, State> {
...
 
-    const fileType = FileType.IDENTIFICATION_PDF;
     const targetUser = this.props.username;
 
+    let fileType;
+    if (
+        userRole === Role.Worker ||
+        userRole === Role.Admin ||
+        userRole === Role.Director
+    ) {
```
Comment (crchong1) [inline src/components/Documents/MyDocuments.tsx:199]: shouldn't workers be able to download identification pdfs as well?

File: src/serverOverride.ts (hunk 1 above)
Comment (steffen12) [reply src/serverOverride.ts:3]: Fixed it now

File: src/components/Documents/MyDocuments.tsx (hunk 3 above)
Comment (steffen12) [reply src/components/Documents/MyDocuments.tsx:199]: Yes, they are able to, but remember that the way that Louis coded it is that the worker "becomes" the client, with the client's username and role, so the checking here is based on that

File: src/components/Documents/UploadDocs.tsx (hunk 2 above)
Comment (steffen12) [reply src/components/Documents/UploadDocs.tsx:158]: Sorry, I think this was blocking us earlier, but the error was unrelated. I've changed it back now

File: src/components/Documents/MyDocuments.tsx (hunk 3 above)
Comment (crchong1) [reply src/components/Documents/MyDocuments.tsx:199]: ok

**Pull request merged**